viewer/packages/360-images/src/Image360Facade.ts

```diff
@@ -21,10 +22,10 @@ export class Image360Facade<T> {
     this._image360Entities.forEach(entity => (entity.icon.hoverSpriteVisible = visible));
   }
 
-  constructor(entityFactory: Image360EntityFactory<T>) {
-    this._entityFactory = entityFactory;
+  constructor(private readonly _entityFactory: Image360EntityFactory<T>) {
```

Comment:
So this `_entityFactory` argument is unneeded?

Reply:
No, it is. Just syntactic sugar for setting it as a field. These two are equivilant:

```ts
private readonly _foo: Bar;
constructor(foo: Bar) {
   this._foo = foo;
  ...
}
```

```ts
constructor(private readonly _foo: Bar) {
  ...
}
```